File: Web/server-api/ServerAPI/budget.core/Models/AccountItem.cs
using budget.core.Services.Interfaces;

namespace budget.core.Models {
  public class AccountItem : Identifiable {
    public required string Title { get; set; }
    public decimal? Initial { get; set; }
    public decimal? Limit { get; set; }
    public decimal? Quantity { get; set; }
    public decimal? Goal { get; set; }
    public string? Comment { get; set; }    
    public required string Currency { get; set; }
    public required AccountType AccountType { get; set; }
  }
}

File: Web/web-app/web-app/src/app/shared/models/account-item.ts
import { AccountType } from "../enums";

export interface AccountItem {
  id?: string,
  title: string;
  initial: number;
	quantity: number;
  limit: number;
	comment:  string;
  accountType: AccountType | undefined;
  currency: string;
  isDeleted: boolean;
  goal: number;
}

